python_configure.bzl

```diff
@@ -391,6 +396,7 @@ python_configure = repository_rule(
     ],
     attrs = {
         "python_version": attr.string(default=""),
+        "python_interpreter_target": attr.label(),
     },
```

Comment:
would suggest library target for completeness

Reply:
Do you mean to substitute the check at `_get_python_lib`? What would this target look like? As far as I understand, we should be getting the correct value by querying the interpreter.